webapp/gunicorn_config.py

```diff
@@ -15,3 +15,4 @@ def child_exit(server, worker):
 workers = 4
 bind = '0.0.0.0:5000'
 worker_tmp_dir = '/tmp'
+access_log_format = '%({x-request-id}i)s %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"'
```

Comment:
Why do we want to include the username `%(u)s`  and what is a potential value for that? I am not really understanding this part.

I am also wondering if we could link to the [access_log_format documentation](https://docs.gunicorn.org/en/stable/settings.html#access-log-format) in a comment or so.

Reply:
> Why do we want to include the username %(u)s and what is a potential value for that? I am not really understanding this part.

This is just the standard access format string copied from the docs. `user` refers to e.g. the username when using Basic authentication.

> I am also wondering if we could link to the access_log_format documentation in a comment or so.

Excellent idea.

Reply:
> This is just the standard access format string copied from the docs. user refers to e.g. the username when using Basic authentication.
Ok. Nevertheless, I am still struggling to understand what is shown in our case because we do not have a username and I was wondering if then the idnr_hash might be shown or something else that we do not want. Maybe, if you do not know, I'll just test it tomorrow.

Reply:
The `user` is the user in the HTTP context, so e.g. if a site is protected by HTTP Basic Auth with username and password, that username will show up in the logs. It does not have access to our application logic and will not log e.g. the idnr_hash

Reply:
The `user` is the user in the HTTP context, so e.g. if a site is protected by HTTP Basic Auth with username and password, that username will show up in the logs. It does not have access to our application logic and will not log e.g. the idnr_hash

Reply:
The `user` is the user in the HTTP context, so e.g. if a site is protected by HTTP Basic Auth with username and password, that username will show up in the logs. It does not have access to our application logic and will not log e.g. the idnr_hash

Reply:
Okay, now I got it. Thanks